Gemfile

```diff
@@ -1,6 +1,8 @@
 source 'https://rubygems.org'
 gemspec
 
+gem 'artifactory', git: 'https://github.com/chef/artifactory-client.git', branch: 'schisamo/build-support'
```

Comment:
@danielsdeleo Yep. Commit message for that change says:

```
[WIP] Point at Artifactory client branch with build support

This will go away when chef/artifactory-client#53 is merged.
```